index.js

```diff
@@ -0,0 +1,23 @@
+const hamburgerButton = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-lists');
+const closed = document.querySelector('.close-icon');
+const mobileList = document.querySelectorAll('.mobile-list');
+
+hamburgerButton.addEventListener('click', (event) => {
+  mobileMenu.style.display = 'block';
+  event.preventDefault();
+});
+
+closed.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+// mobileList.addEventListener('click', () =>{
+//   mobileMenu.style.display = 'none';
+// });
```

Comment:
- [ ] Kindly remove any commented code to make your code clean from any unused code.

Reply:
[OPTIONAL]
- [ ] As requested by the previous review, kindly remove commented code from your code base it make your code unclean